tests/integration/test_app_create.py

```diff
@@ -108,31 +108,44 @@ def test_enforces_secure_headers(self):
             self.assertEqual("1; mode=block", headers["X-Xss-Protection"])
             self.assertEqual("nosniff", headers["X-Content-Type-Options"])
 
-            csp_policy_parts = headers["Content-Security-Policy"].split("; ")
-            self.assertIn("default-src 'self' https://cdn.ons.gov.uk", csp_policy_parts)
-            self.assertIn(
-                f"script-src 'self' https://cdn.ons.gov.uk https://www.googletagmanager.com 'unsafe-inline' 'unsafe-eval' 'nonce-{request.csp_nonce}'",
-                csp_policy_parts,
-            )
-            self.assertIn(
-                "style-src 'self' https://cdn.ons.gov.uk https://tagmanager.google.com https://fonts.googleapis.com 'unsafe-inline'",
-                csp_policy_parts,
-            )
-            self.assertIn(
-                "img-src 'self' data: https://cdn.ons.gov.uk https://www.google-analytics.com https://ssl.gstatic.com https://www.gstatic.com",
-                csp_policy_parts,
-            )
-            self.assertIn(
-                "font-src 'self' data: https://cdn.ons.gov.uk https://fonts.gstatic.com",
-                csp_policy_parts,
-            )
-            self.assertIn(
-                "frame-src https://www.googletagmanager.com", csp_policy_parts
-            )
-            self.assertIn(
-                "connect-src 'self' https://cdn.ons.gov.uk https://cdn.eq.census-gcp.onsdigital.uk",
-                csp_policy_parts,
-            )
+    def test_csp_policy_headers(self):
+        for cdn_url in ["https://cdn.test.domain", "https://cdn.ons.gov.uk"]:
+            with self.subTest(cdn_url=cdn_url):
+                self._setting_overrides = {"EQ_ENABLE_LIVE_RELOAD": False}
+
+                if cdn_url != "https://cdn.ons.gov.uk":
+                    self._setting_overrides["CDN_URL"] = cdn_url
```

Comment:
This isn't clear. It needs to be clearer that the second test is testing `cdn_url=None`